packages/library/README.md

```diff
@@ -401,7 +372,7 @@ const rpc = createRpc({ api, transport });
 
 Note that the `createSolanaRpcApi` function is a wrapper on top of the `createRpcApi` function which adds some Solana-specific transformers such as setting a default commitment on all methods or throwing an error when an integer overflow is detected.
 
-#### Creating your Own API Methods
+#### Creating Your Own API Methods
 
 The new library’s RPC specification supports an _infinite_ number of JSON-RPC methods with **zero increase** in bundle size.
```

Comment:
Nitpick: The unchanged bit below this refers to a custom Quicknode method for Ethereum. Their [Solana docs](https://www.quicknode.com/docs/solana) don't have any equivalent custom methods AFAICT. Might be worth switching out for a Helius link, eg https://docs.helius.dev/compression-and-das-api/digital-asset-standard-das-api/get-asset?